mixins/localize-mixin.js

```diff
@@ -128,6 +136,29 @@ export const LocalizeMixin = dedupeMixin(superclass => class extends superclass
 
 	}
 
+	localizeHTML(key, {
+		_link = '', _tooltip = '', _html = '',
+		_links = {}, _tooltips = {},
+		...replacements
+	} = {}) {
+		const localized = this.localize(key, replacements);
+		const sanitized = localized.replace(/</g, '&lt;').replace(/>/g, '&gt;');
+		const markedUp = sanitized
+			// replace [link], [link key], [/link]
+			.replace(/\[a(?: ?([\w\d-]*))?\]([^]*?)\[\/a\]/g, (m, n, t) => `<d2l-link ${_links[n] || _link}>${t}</d2l-link>`)
+			// replace [tooltip-help], [tooltip-help key], [/tooltip-help]
+			.replace(/\[tooltip-help(?: ?([\w\d-]*))?\]([^]*?)\[\/tooltip-help\]/g, (m, n, t) => `<d2l-tooltip-help inherit-font-style text="${t}">${_tooltips[n] || _tooltip}</d2l-tooltip-help>`)
+			// UNDOCUMENTED: replace [html]
+			.replace('[html]', _html || '')
```

Comment:
I've reduced this down to a simple replace. I think this should remain undocumented for now, and I don't want to encourage overcomplicating langterms with lots of injected HTML. We can expand it later if truly needed.